README.md

```diff
@@ -1,27 +1,50 @@
-# ovh-icon-database-front
+# Ovh Icon Database Front
+
+![ovh-manager-telecom-banner]()
```

Comment:
how about using this banner ? -> https://user-images.githubusercontent.com/3379410/27423240-3f944bc4-5731-11e7-87bb-3ff603aff8a7.png

Reply:
Is it the banner for component, not for a complete ihm ? no ?

Reply:
but if you thing it's OK. i'm Ok.

Reply:
@KamiAmiga done